src/sql/parts/dashboard/dashboard.component.ts

```diff
@@ -36,7 +37,7 @@ export class DashboardComponent extends AngularDisposable implements OnInit {
 	private editDisposable: IDisposable;
 
 	constructor(
-		@Inject(forwardRef(() => DashboardServiceInterface)) private _bootstrapService: DashboardServiceInterface,
+		@Inject(forwardRef(() => CommonServiceInterface)) private _bootstrapService: DashboardServiceInterface,
```

Comment:
Either the injector or the type definition should change in these files. E.g. it should either be 
```
@Inject(forwardRef(() => CommonServiceInterface)) private _bootstrapService: CommonServiceInterface
```
or
```
@Inject(forwardRef(() => DashboardServiceInterface)) private _bootstrapService: DashboardServiceInterface,
```

Reply:
that's the only way I could make it work to use DashboardServiceInterface in dashboard and CommonServiceInterface in other places 

---
In reply to: [182219620](https://github.com/Microsoft/sqlopsstudio/pull/1168#discussion_r182219620) [](ancestors = 182219620)